package.json

```diff
@@ -34,6 +34,7 @@
     "@web3-react/walletconnect-connector": "^6.1.9",
     "@web3-react/walletlink-connector": "6.1.9",
     "bufferutil": "^4.0.5",
+    "date-fns": "^2.29.2",
```

Comment:
do we already have a date formatting library?

Reply:
negative